tutorials/21_render_order.md

```diff
@@ -0,0 +1,52 @@
+\page render_order Render order
+
+This example shows how to set the render order for coplanar poligons.
+
+The material API allows changing the render order. When polygons are coplanar, you can get problems
+with `depth fighting` where the pixels from the two polys compete for the same screen pixel. As you
+can see in the following image:
+
+@image html img/render_order_bad.png
+
+The method `SetRenderOrder` in the Material class allows avoiding this issue. The higher value will
+be rendered on top of other coplanar polygons.
```

Comment:
For the more advanced users, we can add a small explanation letting them know that underneath the hood, we are setting the depth bias value of objects that the material is assigned to.